flowpipe/graph.py

```diff
@@ -506,14 +517,14 @@ def update_node(node, data):
     for name, input_plug in data['inputs'].items():
         node.inputs[name].value = input_plug['value']
         for sub_name, sub_plug in input_plug['sub_plugs'].items():
-            for sub_output in sub_plug['connections'].values():
+            for _ in sub_plug['connections'].values():
```

Comment:
I am a little confused: Why do we loop over the connections, when it does not matter which connection we currently look at? Does this loop no just set `node.inputs[name][sub_name].value` multiple times to the same value?

Reply:
It took me a while to wrap my head around it, but I think you're right, thanks! I changed it